DetectorDescription/DDCMS/plugins/DDCompactViewESProducer.cc

```diff
@@ -48,8 +48,8 @@ class DDCompactViewESProducer : public edm::ESProducer {
 };
 
 DDCompactViewESProducer::DDCompactViewESProducer(const edm::ParameterSet& iConfig)
-    : m_detToken(setWhatProduced(this).consumes<DDDetector>(
-          edm::ESInputTag("", iConfig.getParameter<std::string>("appendToDataLabel")))) {}
+    : m_detToken(setWhatProduced(this).consumes<DDDetector>(edm::ESInputTag("", ""))) {}
+          // edm::ESInputTag("", iConfig.getParameter<std::string>("appendToDataLabel")))) {}
```

Comment:
Empty `ESInputTag` is redundant, so this can be simplified to
```suggestion
    : m_detToken(setWhatProduced(this).consumes<DDDetector>() {}
```
(also the type is nowadays redundant, so this can be further simplified to
```suggestion
    : m_detToken(setWhatProduced(this).consumes() {}
```
)

Reply:
@makortel Thanks. I made this change (but with trailing ')' included). ;-)